resource_activity/models/resource_activity.py

```diff
@@ -661,14 +671,19 @@ def write(self, vals):
                 if 'need_guide' in vals:
                     if not vals.get('need_guide'):
                         vals['guide_product_id'] = False
+                        vals['guides'] = [[6, False, []]]
+                    vals['need_push'] = True

```

Comment:
@houssine78 is this needed? Or should I just do something like in `activity.registrations`:

```python
@api.multi
@api.depends('need_guide', 'guide_product_id', 'guides', ...)
def _compute_push2sale_order(self)
    for activity in self:
        activity.need_push = True  # i.e. for any change
        ...
```